contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -776,12 +779,25 @@ library LiquidationLogic {
         ).collateralizedBalanceOf(params.borrower);
 
         // price of the asset that is used as collateral
-        if (INToken(superVars.collateralXToken).getAtomicPricingConfig()) {
+        if (
+            IXTokenType(superVars.collateralXToken).getXTokenType() ==
+            XTokenType.NTokenUniswapV3
+        ) {
             vars.collateralPrice = IPriceOracleGetter(params.priceOracle)
                 .getTokenPrice(
                     params.collateralAsset,
                     params.collateralTokenId
                 );
+        } else if (
+            IAtomicCollateralizableERC721(superVars.collateralXToken)
```

Comment:
isAtomicToken maybe redundant because it also just check multiplier, need better thinking